submit/main/fund.py

```diff
@@ -84,12 +84,12 @@ def get_active_funds(self, roles: list[str]):
 TOWNS_FUND_APP_CONFIG = FundConfig(
     fund_name="Towns Fund",
     user_role="TF_MONITORING_RETURN_SUBMITTER",
-    current_reporting_period="October to March 2024",
-    current_reporting_round=5,
-    current_deadline=datetime.date(day=28, month=5, year=2024),
+    current_reporting_period="April to September 2024",
+    current_reporting_round=6,
+    current_deadline=datetime.date(day=30, month=11, year=2024),
```

Comment:
In the `reporting_round.csv` file below we have the deadline as 24 November 2024 - here we say 30 November 2024.

Which is correct? Can we be consistent?

Reply:
I believe it's 24th November as Craig mentioned it to @wjrm500 ?